.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Question
+about: Who, what, when, where, and how? 
+
+---
+Hey, stop right there!
```

Comment:
This doesn't feel great but it's a good start. I think we can express this in a way that is more welcoming. Something along the lines of: We want to help you with your question, but this isn't the best place for us to do that. The best place is the forums. We use GitHub to do these other things. Cautious words about what happens if you open an issue here. A lot of this can be done by rearranging what you have already.

Reply:
Also, we can wrap at 80 columns here?